docs/core/project-sdk/msbuild-props.md

```diff
@@ -225,6 +225,8 @@ The `CodeAnalysisTreatWarningsAsErrors` property lets you configure whether code
 
 ### EnforceCodeStyleInBuild
 
+[!NOTE] This feature is currently experimental and may change between the .NET 5 and .NET 6 releases.
```

Comment:
```suggestion
> [!NOTE]
> This feature is currently experimental and may change between the .NET 5 and .NET 6 releases.
```

Reply:
Here's the preview: https://review.docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props?branch=pr-en-us-21274#enforcecodestyleinbuild